api/src/middleware/competenciesRouter.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'express';
```

Comment:
I think we should follow the previous convention and add a line between lines 1 and 2.

Reply:
ok we fixed it.